LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
The license is included in the initial contribution PR, with year and copyright holder specified. So I think you could remove this file from the PR, or fill-on year/fullname and remove it from the other PR.

Reply:
Whoopsiedoodles! I'll fix after #3 is merged, thanks for the catch!